beets/ui/__init__.py

```diff
@@ -471,29 +477,23 @@ def get_replacements():
     return replacements
 
 
-def _pick_format(album, fmt=None):
-    """Pick a format string for printing Album or Item objects,
-    falling back to config options and defaults.
-    """
+def format_(obj, fmt):
+    """Print a object, intended for Album and Item
+
+    This is equivalent to `format`, but the spec can be None
+    `fmt` is mandatory for otherwise one can just call `format(my_object)`"""
```

Comment:
Isn't `format(obj, None)` equivalent to `format(obj, "")` for LibModel objects? That is, the first line in `LibModel.__format__` triggers the default format if the spec is the empty string. This means that `format_(obj)` is the same as `format(obj)`, so is this function necessary?